app/main.py

```diff
@@ -1,8 +1,71 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: float,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car: list) -> float:
+        """
+        Приймає список автомобілів, миє лише ті,
+        у яких clean_mark нижче clean_power станції.
+        Повертає дохід станції від миття цих автомобілів,
+        округлений до 1 десятої.
+        """
+        total_income = 0
+        for i in car:
```

Comment:
```suggestion
        for car in cars:
```
`i` - bad variable name